Source/FieldSolver/FiniteDifferenceSolver/HybridPICModel/HybridPICModel.cpp

```diff
@@ -160,6 +162,13 @@ void HybridPICModel::InitData ()
     const std::set<std::string> resistivity_symbols = m_resistivity_parser->symbols();
     m_resistivity_has_J_dependence += resistivity_symbols.count("J");
 
+    // Adding hyper-resistivity expression - mtobin 20240217
+    m_hyper_resistivity_parser = std::make_unique<amrex::Parser>(
```

Comment:
To track whether to use hyper-resistivity, here we could do:
```suggestion
    m_include_hyper_resistivity_term = (m_eta_h_expression != "0.0");
    m_hyper_resistivity_parser = std::make_unique<amrex::Parser>(
```